### PR TITLE
vera++: increase line length

### PR DESCRIPTION
**Pull request by aabadie**
### Contribution description

The vera++ warning annotation about the 80 character max line length is cluttering the review process on certain files.

This PR is increasing the warning line length to 100 and max line length to 120.

### Testing procedure

The bogus commit should report annotations about maximum line length.

### Issues/PRs references

Should improve the situation in PR like #15718 (see https://github.com/RIOT-OS/RIOT/pull/15718/files#diff-1e6a82d8739f75dd4cddea9ecc25505ed042f3670ee5646dfc0c97dba2881662)

### CODE REVIEW

**Comment by aabadie** [comment]
The new values are correctly taken into account. Next question: do these values make sense ?

**Comment by miri64** [comment]
If the coding conventions are changed, this should be documented as well ;-).

**Comment by miri64** [comment]
> This PR is increasing the warning line length to 100 and max line length to 120.

Are we really needing 120 here? Why not just set he maximum line length to 100, period?

**Comment by bergzand** [comment]
I'm also for a single max. Having a warning line would trigger warning messages from Vera++ on every PR that changes a file with lines over the warning limit, even if the changes are not related to the lines causing the warning (right?)

**Comment by miri64** [comment]
This also would make a mod of the coding conventions unnecessary. The optional 80 char max could still be upheld manually.

**Comment by aabadie** [comment]
> If the coding conventions are changed, this should be documented as well

I pushed b5c7d9d1be to adapt the values in the coding conventions.

**Comment by aabadie** [comment]
Hmm, sorry I misunderstood the comments above. So we agree on a single value of 100 characters per line ?

File: CODING_CONVENTIONS.md [hunk 1]
```diff
@@ -19,8 +19,7 @@
 * Please obey the Linux coding style as described in
   https://www.kernel.org/doc/Documentation/process/coding-style.rst with the
   following modifications and additions:
-    * Line length: aim for no more than 80 characters per line, the absolute
-      maximum should be 100 characters per line.
+    * Line length: aim for no more than 100 characters per line.
```
Comment (miri64) [inline CODING_CONVENTIONS.md:22]: I think this can be kept now as it was.

**Comment by miri64** [comment]
> Hmm, sorry I misunderstood the comments above. So we agree on a single value of 100 characters per line ?

As the absolute maximum, yes. I think however we can keep the 80 char limit as someone an individual developer can look out for or configure their IDE to ;-).

File: CODING_CONVENTIONS.md (hunk 1 above)
Comment (aabadie) [reply CODING_CONVENTIONS.md]: I dropped that change.

**Review by miri64 — APPROVED** [review]
ACK from my side. @jia200x if you like to comment, do it soon ;-).

**Comment by jia200x** [comment]
I'm fine with it!

**Review by maribu — APPROVED** [review]
ACK

**Comment by miri64** [comment]
@aabadie please remove the `REMOVE ME` commit :-).

**Comment by aabadie** [comment]
Another side effect of this PR will be to limit the output of the vera++ check when run on master.

**Comment by miri64** [comment]
> Another side effect of this PR will be to limit the output of the vera++ check when run on master.

I actually thought that was your main motivation :grinning:

**Comment by aabadie** [comment]
> I actually thought that was your main motivation

Let's say it was not the only one :)

**Comment by miri64** [comment]
Mhh for some reason the [80 chars warning still shows up](https://github.com/RIOT-OS/RIOT/runs/1728749196#step:5:27).

**Comment by jia200x** [comment]
> Mhh for some reason the 80 chars warning still shows up.

see https://github.com/RIOT-OS/RIOT/pull/15813

**Pull request merged**